tools/shared.py

```diff
@@ -433,7 +433,10 @@ def set_version_globals():
 
 
 def generate_sanity():
-  return f'{EMSCRIPTEN_VERSION}|{config.LLVM_ROOT}\n'
+  llvm_root = config.LLVM_ROOT
```

Comment:
can you use do `llvm_root = os.path.normcase(config.LLVM_ROOT)` here?

Reply:
Yes, thanks for pointing out